guice/src/main/java/com/flipkart/gjex/grpc/interceptor/FilterInterceptor.java

```diff
@@ -230,6 +235,26 @@ private void configureAccessLog(GrpcFilterConfig grpcFilterConfig,
         }
     }
 
+    private void addAllStaticFilters(GrpcConfig grpcConfig, List<GrpcFilter> filtersForMethod, Map<Class<?>, GrpcFilter> classToInstanceMap) throws ClassNotFoundException {
+        List<String> filterClasses = grpcConfig.getFilterClasses();
+        if (filterClasses != null && !filterClasses.isEmpty()) {
+            for (String filterClass : filterClasses) {
+                try {
+                    Class<?> clazz = Class.forName(filterClass);
+                    if (clazz == AccessLogGrpcFilter.class || clazz.getSuperclass() == AccessLogGrpcFilter.class) {
+                        configureAccessLog(grpcConfig.getGrpcFilterConfig(), filtersForMethod);
```

Comment:
instead of this, have an interface/ by default call the` configure()` method on the filter.

Reply:
Otherwise for your other filters, the config loading would become a problem